Diagnostics/start-windows-build.ps1

```diff
@@ -0,0 +1,405 @@
+    Copy-Item -Path "$DIAGNOSTICS_MESOS_JENKINS_GIT_REPO_DIR\diagnostics\config" -Destination $DIAGNOSTICS_BUILD_BINARIES_DIR -Recurse
+
+    Copy-Item -Force -Path "$DIAGNOSTICS_GIT_REPO_DIR\*.exe" -Destination "$DIAGNOSTICS_BUILD_BINARIES_DIR\"
+    Compress-Files -FilesDirectory "$DIAGNOSTICS_BUILD_BINARIES_DIR\" -Filter "*.*" -Archive "$DIAGNOSTICS_BUILD_BINARIES_DIR\diagnostics.zip"
+    Write-Output "Diagnostic zip file: $DIAGNOSTICS_BUILD_BINARIES_DIR\diagnostics.zip"
+    Write-Output "Diagnostics files were successfully generated"
+}
+function Copy-FilesToRemoteServer {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$LocalFilesPath,
+        [Parameter(Mandatory=$true)]
+        [string]$RemoteFilesPath
+    )
+    Write-Output "Started copying files from $LocalFilesPath to remote location at ${server}:${RemoteFilesPath}"
+    Start-SCPCommand -Server $REMOTE_LOG_SERVER -User $REMOTE_USER -Key $env:SSH_KEY `
+                     -LocalPath $LocalFilesPath -RemotePath $RemoteFilesPath
+}
+
+function New-RemoteDirectory {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$RemoteDirectoryPath
+    )
+    $remoteCMD = "if [[ -d $RemoteDirectoryPath ]]; then rm -rf $RemoteDirectoryPath; fi; mkdir -p $RemoteDirectoryPath"
+    Start-SSHCommand -Server $REMOTE_LOG_SERVER -User $REMOTE_USER -Key $env:SSH_KEY -Command $remoteCMD
+}
+
+function New-RemoteSymlink {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$RemotePath,
+        [Parameter(Mandatory=$false)]
+        [string]$RemoteSymlinkPath
+    )
+    $remoteCMD = "if [[ -h $RemoteSymlinkPath ]]; then unlink $RemoteSymlinkPath; fi; ln -s $RemotePath $RemoteSymlinkPath"
+    Start-SSHCommand -Server $REMOTE_LOG_SERVER -User $REMOTE_USER -Key $env:SSH_KEY -Command $remoteCMD
+}
+function Get-DiagnosticsBuildRelativePath {
+    if($ReviewID) {
+        return "review/$ReviewID"
+    }
+    $repositoryOwner = $GitURL.Split("/")[-2]
+    $mesosCommitID = Get-LatestCommitID
+    return "$repositoryOwner/$Branch/$mesosCommitID"
+}
+
+function Get-RemoteBuildDirectoryPath {
+    $relativePath = Get-DiagnosticsBuildRelativePath
+    return "$REMOTE_DIAGNOSTICS_BUILD_DIR/$relativePath"
+}
+
+function Get-BuildOutputsUrl {
+    $relativePath = Get-DiagnosticsBuildRelativePath
+    return "$DIAGNOSTICS_BUILD_BASE_URL/$relativePath"
+}
+
+function Get-BuildLogsUrl {
+    $buildOutUrl = Get-BuildOutputsUrl
+    return "$buildOutUrl/logs"
+}
+
+function Get-BuildBinariesUrl {
+    $buildOutUrl = Get-BuildOutputsUrl
+    return "$buildOutUrl/binaries"
+}
+
+function New-RemoteLatestSymlinks {
+    $remoteDirPath = Get-RemoteBuildDirectoryPath
+    $baseDir = (Split-Path -Path $remoteDirPath -Parent) -replace '\\', '/'
+    New-RemoteSymlink -RemotePath $remoteDirPath -RemoteSymlinkPath "$baseDir/latest"
+    if($ReviewID) {
```

Comment:
Remove this `if`. I'll make sure the rest of the logic works as expected when testing the build upload later on in the CI.